server/src/main/java/org/elasticsearch/index/engine/SegmentsStats.java

```diff
@@ -144,8 +132,8 @@ public ByteSizeValue getBitsetMemory() {
         return new ByteSizeValue(bitsetMemoryInBytes);
     }
 
-    public ImmutableOpenMap<String, FileStats> getFiles() {
-        return files;
+    public Map<String, FileStats> getFiles() {
```

Comment:
This method appears to be only used in tests. Lets add a comment to that effect so nobody starts using this outright and save ourselves the unmodifiable wrapping?